haskell-style.md

```diff
@@ -391,3 +412,109 @@ import qualified Data.List.NonEmpty as NE
 
 import qualified Data.Sequence as Seq
 ```
+
+### Import groups
+
+Put one blank line between `module`-`where` and the start of your `import`s. Put
+your preferred prelude (when explicit) first, followed by a blank line, then the
+rest of your imports.
+
+Imports are sorted, but with `qualified` versions appearing where they would
```

Comment:
I'm a bit late here, but for import sorting I typically just run `:sort i` on the visual line block of imports in vim. That will group all `qualified` imports together, not interspersed like described here. Can we configure `stylish-haskell` or another code formatter to get this behavior automatically?

Reply:
> I typically just run :sort i on the visual line block of imports in vim. That will group all qualified imports together, not interspersed like described here

This is exactly why I dislike interspersing the `qualified` entries. Now I need to `:sort` and then manually move individual items to get it right.

I think this ship may have sailed, but I'm happy to participate in a discussion around a "one big group + simple sort (so `qualified`s fall together at the end)" option, if you'd want to re-spark it. I may have scared folks off by suggesting categorical grouping at the same time as `qualified`s at end, which made the whole thing feel like a more complex guide.

Reply:
The other option is to use `stylish-haskell` with context-sensitive indenting turned off, which I think will group everything properly.

Then `:! stylish-haskell` over the import lists will work properly.

Reply:
Woah, wait. You can turn off the indentation on `stylish-haskell`? I think that is the solution!

Reply:
PR here: https://github.com/frontrowed/megarepo/pull/1549